roles/validations/tasks/edpm/hotfix.yml

```diff
@@ -0,0 +1,83 @@
+# This job tests the functionality of the dataplane-operator to take an arbitrary hotfixed
+# image as input, then role out the hotfix to the deployed nodes.
+#
+# This job was created to satisfy:
+# https://issues.redhat.com/browse/osprh-5326
+
+# Since jobs use different names for their OpenStackDataPlaneNodeSet's, we need to dynamically
+# determine the name of the currently deployed one
+- name: Determine name of deployed NodeSet
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  cifmw.general.ci_script:
+    output_dir: "{{ cifmw_openshift_login_basedir }}/artifacts"
```

Comment:
so you'll have to update each occurrence of `cifmw_openshift_login_basedir` to `cifmw_validations_basedir`